lib/datastore/datastore_importer.es6.js

```diff
@@ -0,0 +1,145 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+require('../web_catalog/api_extractor.es6.js');
+require('../web_apis/browser.es6.js');
+require('../web_apis/web_interface.es6.js');
+require('../web_apis/browser_interface_relationship.es6.js');
+require('../web_apis/api_importer.es6.js');
+require('./rate_limited_DAO.es6.js');
+const objectGraph = require('object-graph-js').ObjectGraph;
+const fs = require('fs');
+
+foam.CLASS({
+  name: 'DatastoreImporter',
+  package: 'org.chromium.apis.web',
+  documentation: `Google Cloud Datastore importer handles importing
+      browserAPIs from object graph files to a Google Cloud Datastore.`,
+  imports: ['info'],
+  requires: [
+    'org.chromium.apis.web.ApiExtractor',
+    'org.chromium.apis.web.ApiImporter',
+    'com.google.cloud.datastore.DatastoreDAO',
+    'org.chromium.apis.web.RateLimitedDAO',
+    'org.chromium.apis.web.BrowserWebInterfaceJunction',
+    'org.chromium.apis.web.Browser',
+    'org.chromium.apis.web.WebInterface',
+  ],
+  properties: [
+    {
+      name: 'apiExtractor',
+      documentation: `Extracts interface catalog from object
+          graph object.`,
+      factory: function() {
+        return this.ApiExtractor.create();
+      },
+    },
+    {
+      name: 'apiImporter',
+      documentation: `Imports interface catalog to cloudStore DAO.`,
+      factory: function() {
+        return this.ApiImporter.create({
+          toCloudDatastore: true,
```

Comment:
I don't think this exists anymore. The documentation above should probably also be updated to not mention cloud datastore, since that's an implementation detail of init.